webclient/http1/src/main/java/io/helidon/webclient/http1/Http1CallChainBase.java

```diff
@@ -508,6 +508,9 @@ private void ensureBuffer() {
             int length;
             try {
                 length = Integer.parseUnsignedInt(hex, 16);
+                if (length < 0) {
+                    throw new IllegalArgumentException("Chunk size cannot be negative " + length);
```

Comment:
Hmm; the message seems wrong. If the user specifies `ffffffff`, she is specifying a positive number. This message implies she is specifying a negative one, and does not report the original input. That said I'm not sure what should go here.

Reply:
I see your point, but interpreted as an int in complement 2, it is a negative. Working a different approach anyway.